src/main/java/racing/InputView.java

```diff
@@ -24,9 +28,32 @@ public int inputtedNumber(String message) {
 
     }
 
+    public List<String> inputtedCarNameList(String message) {
+        System.out.println(message);
+        String inputtedCarNames = scanNextLine();
+
+        List<String> carNameList = StringUtil.splitStringToList(inputtedCarNames, CAR_NAME_DELIMITER);
+
+        if (isCarNameLengthOverStandard(carNameList)) {
+            return inputtedCarNameList(CAR_NAME_LENGTH_OVER);
+        }
+
+        return carNameList;
+    }
+
+
+
+    private boolean isCarNameLengthOverStandard(List<String> carNameList) {
```

Comment:
불필요한 공백은 제거해 주시면 좋을 것 같아요
```suggestion

    private boolean isCarNameLengthOverStandard(List<String> carNameList) {
```